holoviews/plotting/mpl/renderer.py

```diff
@@ -318,3 +321,4 @@ def load_nb(cls, inline=True):
         """
```

Comment:
I agree that a property would involve an underscore attribute then the property itself (public API). I don't mind really, a public attribute is fine too.